pysidt/sidt.py

```diff
@@ -346,7 +346,7 @@ class MultiEvalSubgraphIsomorphicDecisionTree(SubgraphIsomorphicDecisionTree):
         `decomposition`: method to decompose a molecule into substructure contributions.
         `root_group`: root group for the tree
         `nodes`: dictionary of nodes for the tree
-        `n_strucs`: number of splits to try for each node. Default is 1.
+        `n_strucs`: number of disconnected structures that should be in the group. Default is 1.
```

Comment:
Hmm...so suppose you have an intra_H_migration we could conceivably start with an R. and separate a R-H that eventually become the same molecule...since we only create bonds in this process (never destroy bonds) it might be best to say this is the "minimum number of disconnected structures that can be in the group".

Reply:
I change `n_strucs` to `n_strucs_min` to reflect that it's the minimum